Extraction demo

stevengans opened the pull request:
Extract.py is a script that can be used to extract specified annotated images from a specified book from the manga109 dataset.
- Readme and output example included

stevengans committed:
Extraction demo with readme

Extract.py is a script that can be used to extract specified annotated images from a specified book from the manga109 dataset.
- Added a readme
- Added an output example like visualization demo

stevengans committed:
Merge pull request #1 from stevengans/extraction-demo

Extraction demo with readme

woodrush commented:
Thanks, I tested the code and saw it working. Before merging, could you do the following fixes?

- Please change the following line; I think this is an artifact from when it was from manga109api, since the code ran in this directory without putting it in the manga109api repo.
  > please make sure this script is in the manga109api directory along with the dataset.
- Please add the following line below out.png for copyright attribution, following https://github.com/manga109/manga109-demos/blob/master/visualization/README.md :
  ```
  Yume no Kayoiji, (c) Yasuyuki Oono
  ```
  This is as per Manga109's usage instructions (http://www.manga109.org/en/download.html),
  > When using parts of the dataset within academic papers or videos, please note the author’s copyright notice as “© Author’s Name” and note that the work was cited from the Manga109 dataset.

Otherwise, LGTM!

matsui528 commented on extraction/README.md:
````diff
@@ -0,0 +1,18 @@
+
+When using the code, please make sure this script is in the manga109api directory along with the dataset. For a usage example, if you want to extract a faces from 5 pages of YumeNoKayoiji's book, please run the following command : 
+
+```python
````
Just a minor thing. This should be ` bash` or `console`, not `python`

woodrush replied:
@stevengans Could you change this to `bash` too?

stevengans replied:
@matsui528 can you clarify the ask. Do you want this as a .sh file instead of a .py file?

woodrush replied:
I think @matsui528 's commenting on the
```
 ```python
 python extract.py --book YumeNoKayoiji --annotation face --pages 5 --preprocess --size 256
```
part. Since this is executed in a shell like bash, for proper syntax highlighting, the
```
 ```python
```
should be changed to:
```
 ```bash
```

stevengans replied:
@woodrush Done

woodrush replied:
Thanks!

stevengans committed:
Updated README.md

- Added  copyright attribution
- Edited out the old requirement of being inside the api directory.

stevengans committed:
Changed python to bash

woodrush commented:
LGTM! Thanks for the PR.
